metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py

```diff
@@ -1152,13 +1152,10 @@ def _get_looker_dashboard(self, dashboard: Dashboard) -> LookerDashboard:
         return looker_dashboard
 
     def _get_looker_user(self, user_id: Optional[str]) -> Optional[LookerUser]:
-        user = (
-            self.user_registry.get_by_id(
-                user_id,
-            )
-            if self.source_config.extract_owners and user_id is not None
-            else None
-        )
+        if user_id is None:
+            return None
+
+        user = self.user_registry.get_by_id(user_id)
```

Comment:
Is it safe to remove ` self.source_config.extract_owners` check here ? I'm afraid that would ingest owners even if the flag is not set .

How about calling LookerSDK's `all_users` api in LookerUserRegistry init and using that info to populate platform resource as well as first level cache for `get_by_id`.

Reply:
Resolved